src/components/victory-stack/victory-stack.js

```diff
@@ -413,18 +424,33 @@ export default class VictoryStack extends React.Component {
     return React.cloneElement(containerComponent, parentProps);
   }
 
-  renderGroup(children, style) {
+  renderGroup(children, modifiedProps, style) {
+    const { clipPathComponent } = modifiedProps;
+    const barWidth = style.data.width;
+    const clipComponent = React.cloneElement(clipPathComponent, Object.assign(
+      {},
+      {
+        padding: modifiedProps.padding,
+        clipId: modifiedProps.clipId,
+        barWidth,
+        clipWidth: modifiedProps.clipWidth + barWidth * 2 || modifiedProps.width + barWidth * 2,
```

Comment:
this is fixed in this commit https://github.com/FormidableLabs/victory-chart/pull/333/commits/99e7af3b863494df1eaf5a38da7cc35390ed3d59